src/main/dialogs/psp.js

```diff
@@ -430,6 +434,27 @@ gpii.app.psp.closePSP = function (psp, settingsBroker) {
     });
 };
 
+/**
+ * Focuses back the QSS when the PSP is closed. Note that if the PSP has been closed
+ * as a result of a keyboard interaction, the "Sign in" button in the QSS should be
+ * highlighted. Thus, this function should internally use `qss.show` instead of
+ * `qss.focus`.
+ * @param {Component} qssWrapper - The `gpii.app.qssWrapper` instance.
+ * @param {Object} params - Parameters specifying how the `onClosed` event was
+ * triggered (e.g. which keyboard key if any resulted in it).
+ */
+gpii.app.psp.giveFocusToQss = function (qssWrapper, params) {
```

Comment:
we should probably use the same method also for the QSS Widget dialog

Reply:
What exactly do you mean?